File: php/clockhands.php
<?php
    for ($i = 0 ; $i < 11 ; $i++){
        $t = floor((43200 * $i + 21600) / 11);
        $h = floor($t / 3600);
        $m = floor($t / 60) % 60;
        $s = $t % 60;
        echo sprintf("%02d:%02d:%02d\n", ($h == 0) ? 12 : $h, $m, $s);
    }
?>

File: matlab/clockhands.m
t = ((0:10) + 0.5) * 43200 / 11;
h = floor(t / 3600) + 12 * (floor(t / 3600) == 0);
m = floor(mod(t / 60, 60));
s = floor(mod(t, 60));
fprintf('%02i:%02i:%02i\n',[h; m; s])